constraint/pkg/client/regolib/src.go

```diff
@@ -4,8 +4,14 @@ const (
 	targetLibSrc = `
 package hooks["{{.Target}}"]
 
+# Auto reject reviews when a constraint has namespaceSelector but Namespace is not cached
+deny[response] {
```

Comment:
The format for `response` should be handled by this Rego package, as the CF relies on the interface defined by response to parse/return violations.

Reply:
so maybe `autoreject_review` should just return the msg?